testkit-backend/backend.go

```diff
@@ -1379,15 +1382,33 @@ func getAuth(authTokenMap map[string]any) (neo4j.AuthToken, error) {
 	case "bearer":
 		authToken = neo4j.BearerAuth(authTokenMap["credentials"].(string))
 	default:
-		parameters := authTokenMap["parameters"].(map[string]any)
-		if err := patchNumbersInMap(parameters); err != nil {
-			return neo4j.AuthToken{}, err
+		var parameters map[string]any
+		if v, ok := authTokenMap["parameters"].(map[string]any); ok {
+			parameters = v
+			if err := patchNumbersInMap(parameters); err != nil {
+				return neo4j.AuthToken{}, err
+			}
+		}
+
+		var scheme, principal, credentials, realm string
+		if v, ok := authTokenMap["scheme"].(string); ok {
+			scheme = v
+		}
```

Comment:
Refactoring potential: 4 times basically the same 3 lines. Something along the lines of

```go
func mapGetString(data map[string]any, key string) string {
	out, _ := data[key].(string)
	return out
}
```